src/token/mod.rs

```diff
@@ -370,10 +384,16 @@ fn partial_tokens_to_tokens(mut tokens: &[PartialToken]) -> EvalexprResult<Vec<T
                                     cutoff = 3;
                                     Some(Token::Float(number))
                                 } else {
-                                    Some(Token::Identifier(literal.to_string()))
+                                    return Err(EvalexprError::IllegalIdentifierSequence(
```

Comment:
Here we are parsing the literal as a number, so this should be a new error variant like `IllegalNumericLiteral`.

Reply:
Ok. I will change this